client/src/pages/people/Form.js

```diff
@@ -17,6 +17,8 @@ import {Person} from 'models'
 
 import CALENDAR_ICON from 'resources/calendar.png'
 
+const ADVISOR = 'ADVISOR'
+const PRINCIPAL = 'PRINCIPAL
```

Comment:
Missing a quote?

Reply:
Indeed, and I guess these const's could go somewhere public and be used everywhere the person.role is now a hard-coded string (e.g. on line 68 below).